releasenotes/notes/4500-barrier-counted-as-non-local-gate-6cb31d63a9052632.yaml

```diff
@@ -0,0 +1,7 @@
+---
+fixes:
```

Comment:
Since this modifies previous behavior on user-land API, maybe `upgrade` section?

Reply:
In this case, the old behavior (including barriers in the count) was wrong, so I went with fixes, but I don't have a strong opinion on it.